predict_detectron2.py

```diff
@@ -129,7 +131,7 @@ def prediction(self, image_path):
 
     def get_class_to_category(self):
         class_to_category = {}
-        with open("utils/class_to_category.json") as fp:
+        with open("utils/class_to_category_round2.json") as fp:
```

Comment:
I think you need to upload `class_to_category_round2.json`, while you changed `class_to_category.json` instead.